src/tags/visual/Collapse.js

```diff
@@ -55,6 +55,7 @@ const PanelModel = types.model({
     'audio',
     'image',
     'hypertext',
+    'tabletext',
```

Comment:
These are required to allow tabletext to be one of the child tag of the `<View/>` or collapsable `<Panel/>